deploy-pkg/webhook-broker-chart/README.md

```diff
@@ -12,3 +12,24 @@ broker:
 ```
 
 Depending on how you want the application to react pass a configuration; by default on any config map change the existing pod processes will stop, if its `restart`, the web endpoint and workers will restart and if its `ignore`, then the application will not watch for config file change and hence won't react either.
+
+## DB Pruning
+Webhook Broker writes pruned messages to a local directory alongside GCS/S3 storage if `broker.dbPruning.enabled` is set to `true`. This local output is intended strictly for testing and debugging purposes. Please note that local files exist only on the node where the pod is running and should not be relied upon for production durability or backup. To avoid requiring cloud-specific storage classes, we use a Kubernetes hostPath volume.
+
+```yaml
+broker:
+  dbPruning:
+    enabled: true
+    cronSchedule: "*/10 * * * *"
+    pvStorage: 32Gi
+    pvReclaimPolicy: Delete
+    timeoutSeconds: 1800
+    config: |
+      [prune]
+      export-node-name=webhook-broker
+      message-retention-days=30
+      export-path=/mnt/ephemeral
+      remote-export-destination=s3
+      remote-export-url=s3://example-bucket/
```

Comment:
We can remove the trailing `/`, it's automatically added when file paths are constructed.
```suggestion
      remote-export-url=s3://example-bucket
```